scripts/build/utils/utils.js

```diff
@@ -38,7 +38,19 @@ function getRuleCSS({ styleLoader, localIdentName, includePaths, context }) {
     };
 }
 
+function getRuleFiles({ fileLoader }) {
+    return {
+        test: /\.(png|jpe?g|gif)$/i,
```

Comment:
Adding svg is a good point. But having it already defined for other img types had no negative effect, as far as I know. But it will make things smoother if we ever happen to add something like that. Or am I missing something?